system-test/spanner.ts

```diff
@@ -6144,6 +6144,186 @@ describe('Spanner', () => {
       });
     });
 
+    describe('dml returning', () => {
```

Comment:
i feel dml returning should be a subsection under dml and not separate, you can have a describe inside a describe as well

Reply:
Done